pkgs/development/python-modules/cut-cross-entropy/default.nix

```
@@ -0,0 +1,97 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  pythonAtLeast,
+
+  # build-system
+  setuptools,
+  setuptools-scm,
+
+  accelerate,
```

Can you please be consistent? If you add a header `# build-system`, can you please add a header `# dependencies`. Or simply remove it.

```suggestion
  # dependencies
  accelerate,
```